src/shapes/Text/Text.ts

```diff
@@ -773,9 +784,9 @@ export class Text<
    */
   _measureChar(
     _char: string,
-    charStyle: TextStyleDeclaration,
+    charStyle: CompleteTextStyleDeclaration,
     previousChar: string | undefined,
-    prevCharStyle: any
+    prevCharStyle: CompleteTextStyleDeclaration | Record<string, never>
```

Comment:
what does this mean?
Or a style or any empty object?

Reply:
yes. or complete or empty.
Not sure why, i just satisfied it.